README.md

```diff
@@ -31,7 +31,7 @@ Profiling in a production environment is not recommend.
   ddev restart
   ```
 
-- Install a profiler. If your application uses composer, you can install it with
+- Install `perftools/php-profiler`
```

Comment:
Would it be possible/appropriate to do this in a `config.xhgui.yaml` post-start hook? It's posslble that would do harm, but it would be nice for it to be packaged up better.

Reply:
Do people use other profilers? 
Could we be creating composer issues?

Reply:
I don't know. The advantage of the way you have it right now is that people have to take that step by themselves. The disadvantage is that people don't expect add-ons to require several steps of extra work.

Reply:
See [other comment.](https://github.com/tyler36/ddev-xhgui/pull/5#discussion_r1386090645) .

There seems to be interest in xhgui. I'm sure others will let us know what they prefer.